cmd/juno/juno.go

```diff
@@ -207,8 +229,33 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 
 		// TextUnmarshallerHookFunc allows us to unmarshal values that satisfy the
 		// encoding.TextUnmarshaller interface (see the LogLevel type for an example).
-		return v.Unmarshal(config, viper.DecodeHook(mapstructure.ComposeDecodeHookFunc(
-			mapstructure.TextUnmarshallerHookFunc(), mapstructure.StringToTimeDurationHookFunc())))
+		if err := v.Unmarshal(config, viper.DecodeHook(mapstructure.ComposeDecodeHookFunc(
+			mapstructure.TextUnmarshallerHookFunc(), mapstructure.StringToTimeDurationHookFunc()))); err != nil {
+			return err
+		}
+
+		// Set custom network
+		if v.IsSet(cnNameF) {
+			fallBackSequencerAddress, _ := new(felt.Felt).SetString("0x046a89ae102987331d369645031b49c27738ed096f2789c24449966da4c6de6b")
+			l1ChainID, ok := new(big.Int).SetString(v.GetString(cnL1ChainIDF), 0)
+			if !ok {
+				return fmt.Errorf("invalid L1 chain id %s", v.GetString(cnL1ChainIDF))
+			}
+			config.Network = utils.Network{
+				Name:                v.GetString(cnNameF),
+				FeederURL:           v.GetString(cnFeederURLF),
+				GatewayURL:          v.GetString(cnGatewayURLF),
+				L1ChainID:           l1ChainID,
+				L2ChainID:           v.GetString(cnL2ChainIDF),
+				CoreContractAddress: common.HexToAddress(v.GetString(cnCoreContractAddressF)),
+				BlockHashMetaInfo: &utils.BlockHashMetaInfo{
+					First07Block:             0,
+					FallBackSequencerAddress: fallBackSequencerAddress,
```

Comment:
Custom networks shouldn't have fallback sequencer address

Reply:
I've removed it, and updated the VerifyBlockHash func